examples/substreams-powered-subgraph/README.md

```diff
@@ -0,0 +1,25 @@
+# Substreams-powered subgraph: tracking contract creation
+
+A basic Substreams-powered subgraph, including the Substreams definition. This example detects new
+contract deployments on Ethereum, tracking the creation block and timestamp. There is a
+demonstration of the Graph Node integration, using `substreams_entity_change` types and helpers.
+
+## Prerequisites
+
+This
+[requires the dependencies necessary for local Substreams development](https://substreams.streamingfast.io/developers-guide/installation-requirements),
+as well as Graph CLI >=0.51.1:
+
+```
+npm install -g @graphprotocol/graph-cli
```

Comment:
we should create a simple `package.json` file and add CLI as a dependency in it. I think we can even call the make scripts from the `package.json` then we can just have one `build` script that in `postbuild` calls Substreams CLI to build and package

Reply:
I like this, done, I think that makes for a cleaner experience